api/src/main/java/org/openmrs/module/fhir/api/util/FHIRPersonUtil.java

```diff
@@ -154,18 +154,17 @@ public static Person generatePerson(org.openmrs.Person omrsPerson) {
 	 * @should generate Oms Person
 	 */
 	public static org.openmrs.Person generateOpenMRSPerson(ca.uhn.fhir.model.dstu2.resource.Person personFHIR) {
-
 		org.openmrs.Person omrsPerson = new org.openmrs.Person();
-
+		omrsPerson.setUuid(personFHIR.getId().getIdPart());
```

Comment:
what happen if request not contain id part? This may lead to null pointer. Shall we check that